model_types/modelTypesList.json

```diff
@@ -0,0 +1,90 @@
+{
+  "modelTypes": [
+    {
+      "modelType": "synthetics",
+      "defaultConfig":  "config_templates/gretel/synthetics/tabular-lstm-evaluate.yml",
```

Comment:
@danielleali we'll need a `description` and icon source as well, right?

Reply:
no, actually @thesteady already hardcoded the icon, label and description for all the model types in console, so we don't need to have that again.